test/unit_tests/kinsol/CXX_serial/kin_test_getjac.out

```diff
@@ -0,0 +1,14 @@
+nni = 20
+Jac nni = 11
+
+   Index                     J DQ                   J true      absolute difference      relative difference
+------------------------------------------------------------------------------------------------------------
+       0    3.000000000000000e+00    3.000000000000000e+00    0.000000000000000e+00    0.000000000000000e+00
+       1    9.944679290056229e-01    9.962893670467480e-01    1.821438041125112e-03    1.828221901558893e-03
+       2    2.687990665435791e-01    2.204736235493865e-01    4.832544299419261e-02    2.191892264308326e-01
```

Comment:
I am rather surprised by the large differences between "J DQ" and "J true" here.  Do you know why these are NOT closer to sqrt(unit_roundoff)?

Reply:
The large difference is because "J DQ" is evaluated at iteration 11 while "J true" is evaluated at the final solution. When "J DQ" is evaluated closer the final iterate the differences are closer to sqrt(unit_roundoff). I can modify the test test setup so "J DQ" is evaluated more frequently.